### PR TITLE
chore(commitlint): disable body-max-line-length

### PR DESCRIPTION
**Pull request by WikiRik**
This is done so the auto-generated dependabot PRs won't fail the commitlint check

### CODE REVIEW

File: .commitlintrc.json [hunk 1]
```diff
@@ -1 +1,6 @@
-{ "extends": ["@commitlint/config-conventional"] }
+{
+    "extends": ["@commitlint/config-conventional"],
+    "rules": {
+        "body-max-line-length": [0]
+    }
+}
```
Comment (ephys) [inline .commitlintrc.json:6]: Other JSON files in this project use 2 spaces per indentation level

**Review by ephys — COMMENTED** [review]
A broader question about this: Do we want to align how PR are managed with the other Sequelize repos?

That would mean:
- Squash commits only
- Commit messages don't matter, only the PR title is linted (pending on our other repos too)

**Comment by WikiRik** [comment]
Yes, that and things like the usage of dependabot vs renovate are things that we should discuss with the original team. We could take over fully and align our processes/styling with the main repo but I can imagine that not everybody will be that happy with that

**Comment by lukashroch** [comment]
Alignment with sequelize sounds reasonable, imo it's not worth to fraction the workflows. Personally, I prefer renovate anyway. Though I like to group deps and let it run on weekly schedule or something ... these per-one-dependency commits just clutter the history too much for non critical deps 🙂

**Review by lukashroch — APPROVED** [review]

**Pull request merged**